apps/ledger-live-desktop/tests/page/modal/send.modal.ts

```diff
@@ -52,13 +52,19 @@ export class SendModal extends Modal {
     await this.fillRecipientInfo(tx);
     await this.continueButton.click();
     await this.cryptoAmountField.fill(tx.amount);
-    await this.feeStrategy(tx.speed).click();
+    if (
+      tx.accountToDebit.currency == Currency.sepETH &&
+      tx.accountToDebit.currency == Currency.POL &&
+      tx.accountToDebit.currency == Currency.DOGE &&
+      tx.accountToDebit.currency == Currency.BCH
```

Comment:
not sure to understand that condition, you expect currency to be `sepETH` AND to be `POL` AND to be `DOGE` AND to be `BCH`, correct?
shouldn't it be `||` ?

Reply:
on top of what @bharamboure-ledger says having so many conditions in the "if" seems a bit dirty to be honest